.github/workflows/npm-cd.yml

```diff
@@ -327,26 +306,44 @@ jobs:
                   npm-auth-token: ${{ secrets.NPM_AUTH_TOKEN }}
                   arch: ${{ matrix.build.ARCH }}
 
-            - name: Checkout
-              if: ${{  matrix.build.TARGET != 'aarch64-unknown-linux-musl'}}
-              uses: actions/checkout@v4
-              with:
-                  submodules: "true"
-
-            - name: Install node
-              if: ${{ !contains(matrix.build.TARGET, 'musl') }}
+            - name: Setup node
+              if: ${{ matrix.build.TARGET != 'aarch64-unknown-linux-musl' }}
               uses: actions/setup-node@v3
               with:
                   node-version: "16"
                   registry-url: "https://registry.npmjs.org"
+                  architecture: ${{ matrix.build.ARCH }}
                   scope: "${{ vars.NPM_SCOPE }}"
                   always-auth: true
+                  token: ${{ secrets.NPM_AUTH_TOKEN }}
+
+            - name: Install tsc and compile utils
+              shell: bash
+              working-directory: ./utils
+              run: |
+                  npm install
+                  npm install -g typescript
```

Comment:
can't it be a dependency in the package.json file?

Reply:
I need it global